src/app/@ansyn/map-facade/components/ansyn-popover/ansyn-popover.component.ts

```diff
@@ -9,6 +9,8 @@ export class AnsynPopoverComponent {
 
 	@Input() readonly icon: string;
 
+	@Input() hideOverflow: boolean;
```

Comment:
Very nice. Just the name should be reversed, like dontHIdeOverflow, or someting like that